Documentation/data-collection.md

```diff
@@ -991,6 +991,30 @@ data:
     #
     # apiserver_list_watch_request_success_total:rate:sum represents the rate of change for successful LIST and WATCH requests over a 5 minute period.
     - '{__name__="apiserver_list_watch_request_success_total:rate:sum", verb=~"LIST|WATCH"}'
+    #
+    # owners: (https://github.com/stackrox/stackrox, @stackrox/eng)
+    #
+    # rhacs:telemetry:rox_central_secured_clusters provides the number of clusters secured by a Central instance of a
+    # Red Hat Advanced Cluster Security installation.
+    - '{__name__="rhacs:telemetry:rox_central_secured_clusters"}'
```

Comment:
(suggestion) Looking at the `rhacs:telemetry:rox_central_secured_clusters` expression:

```
max by (build, central_id, central_version, hosting, install_method) (                                                                                                      
  rox_central_secured_clusters{branding=~"rhacs|RHACS"}                                                                                                                     
)
```

I'd recommend to normalize the branding value in the recorded metric, either when producing the metric (e.g. always upper-case the label value) or in the rule expression using label_replace().